src/app/loadout-builder/generated-sets/GeneratedSet.m.scss

```diff
@@ -26,31 +27,21 @@
   }
 }
 
-.existing-loadout {
-  margin: 0;
-  color: #e8a534;
-}
-
-.loadout-name {
-  font-weight: bold;
-}
-
 .items {
   display: flex;
   flex-direction: row;
   flex-wrap: wrap;
-  justify-content: flex-start;
+  justify-content: space-around;
   padding: 0;
+  max-width: 800px;
 
-  @include phone-portrait {
-    flex-direction: column;
+  // Prevents odd number of columns messing up the layout
+  @media (min-width: 780px) and (max-width: 1079px) {
```

Comment:
Can we maybe use CSS grid to avoid having to make breakpoints?

Reply:
I couldn't get grid to make a reliable 2/3/6 column split, only a 2/6.

Reply:
Can we use the auto placement stuff for that?